Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestCaseDeprioritizedEvent.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Autopsy Forensic Browser
+ *
+ * Copyright 2018 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.experimental.autoingest;
+
+import java.io.Serializable;
+import org.sleuthkit.autopsy.events.AutopsyEvent;
+
+/**
+ * Event published when an automated ingest manager deprioritizes all or part of
+ * a case.
+ */
+final class AutoIngestCaseDeprioritizedEvent extends AutopsyEvent implements Serializable {
```

Comment:
It looks like you forgot to add event handlers for these new events in AutoIngestManager.propertyChange() and AutoIngestMonitor.propertyChange(). But I would lean towards re-using AutoIngestCasePrioritizedEvent instead of making a new event in the first place, mostly because it will reduce the chance of errors like this. The handling of the two events is exactly the same and I can't see a case where a remote node would care whether a case was prioritized or de-prioritized; either way we need to rescan the directory.